content/en/system_config/installation.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Installation"
-category: "Cosign"
-position: 102
+category: "System configuration"
```

Comment:
I think the System Configuration section should be higher up, because installation is the first step. I would recommend it's after "About Sigstore" and before "Signing"

Reply:
We were thinking that you'd only install and configure once, but I can move  it.

Reply:
Another option could be a link in the first page of Signing that says "as a prereq, install cosign" and link to later documentation?

Reply:
> Another option could be a link in the first page of Signing that says "as a prereq, install cosign" and link to later documentation?

The Getting Started page already says this:

Getting Started (Quick Start)
Cosign is a command line utility that can sign and verify software artifact, such as container images and blobs. Join us on our [Slack channel](https://sigstore.slack.com/). (Need an [invite](https://links.sigstore.dev/slack-invite)?)

Installation
To sign software artifacts and verify signatures using Sigstore, you need to install Cosign. Instructions to install Cosign can be found on the [Cosign Installation page](https://github.com/jonvnadelberg/sigstore-docs/blob/update-sigstore-copy/system_config/installation). This will allow you to sign and verify both blobs and containers.

Is that acceptable?

Reply:
Yea, that looks sufficient. 

I'd recommend that Policy Controller and this section be moved above Certificate Authority and Transparency Log, given the latter two are for developers only.

Reply:
ok